front-end/package-lock.json

```diff
@@ -78,7 +78,6 @@
             "integrity": "sha512-sEyWjw28a/9iluA37KLGu8vjxEIlb60uxznfTUmXImy7H5NvbpSO6yYgmgH5KiD7j+zTUUihiST0jEP12IoXow==",
             "dev": true,
             "license": "MIT",
-            "peer": true,
```

Comment:
I don't think there were any changes to package.json, so we don't need package-lock.json in this diff